src/ontology/cl-edit.owl

```diff
@@ -7737,11 +7737,13 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GO:tfm") obo:IAO_0000115 obo:
 AnnotationAssertion(rdfs:label obo:CL_0000511 "androgen binding protein secreting cell")
 SubClassOf(obo:CL_0000511 obo:CL_0000154)
 
-# Class: obo:CL_0000512 (paracrine cell)
+# Class: obo:CL_0000512 (OBSOLETE paracrine cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0198529171") obo:IAO_0000115 obo:CL_0000512 "A paracrine cell is a secretory cell that secretes an agonist into intercellular spaces in which it diffuses to a target cell other than that which produced it.")
-AnnotationAssertion(rdfs:label obo:CL_0000512 "paracrine cell")
-SubClassOf(obo:CL_0000512 obo:CL_0000151)
+AnnotationAssertion(oboInOwl:consider obo:CL_0000512 "secretory cell")
```

Comment:
I think David meant to add the CL term for secretory cell (CL:0000151) in the consider annotation :)